drivers/SmartThings/matter-lock/fingerprints.yml

```diff
@@ -10,9 +10,15 @@ matterManufacturer:
     vendorId: 0x135d
     productId: 0xb0
     deviceProfileName: lock-nocodes-notamper
+#Aqara
+  - id: "4447/10242"
+    deviceLabel: Aqara Smart Lock U200
+    vendorId: 0x115F
+    productId: 0x2802
+    deviceProfileName: lock-nocodes-nobattery-lockAlarm
```

Comment:
@HunsupJung Is the intention to later add the new lock code capability to this profile so that existing devices automatically get lock code support? If so I'd suggest not naming the profile "nocodes"

Reply:
@tpmanley , @ctowns 
If the same profile is used afterwards, it seems correct to remove nocodes from the name. How about lock-lockalarm-nobattery that simply removes nocodes from the name?

Reply:
[d0a77c9](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1413/commits/d0a77c936e3ea9e7dfbf99220f3f0fa63051754b) Updated this to be `lock-lockalarm-nobattery` but NOT include lock codes in the profile capabilities so that we can add the new lock codes capability at a later date but still use the same profile name.